CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ project(acquire-driver-zarr)
 cmake_policy(SET CMP0079 NEW) # allows use with targets in other directories
 enable_testing()
 
+find_package(nlohmann_json CONFIG REQUIRED)
+find_package(blosc CONFIG REQUIRED)
```

Comment:
We like to keep the base CMakeLists.txt pretty clean. As the list of dependencies grows, this could become pretty cluttered. Would you mind putting this into the `cmake` folder, in `dependencies.cmake` or something like that?

Reply:
- For small to medium projects, I typically choose to place `find_package` commands directly in the main CMakeLists.txt file. This makes it easier to understand the dependencies at a glance, slightly reducing the overhead of understanding the project structure.
- Creating separate CMake modules can help keep the main CMakeLists.txt file tidy. In the past, I created them mainly for reusability. However, I don't believe this is a particularly good example of a reusable component.
- Ideally, we keep our dependency list small.

I don't think it's a big deal either way, so if you feel strongly about this, I'm happy to move it to a separate module.